### DISCUSSION TITLE
Zipato RFID unsolicited notification

### DISCUSSION THREAD

**Discussion by duylong** [zwave-js/zwave-js#6573] [Feature Requests]
### Description

Hi,

I use Zipato MINI KEYPAD RFID to activate or deactivate scenarios. Each time the module wakes up, it refreshes the values as if I had used my RFID key, which unintentionally triggers my scenario

```
12:26:31.214 CNTRLR « [Node 044] received wakeup notification
12:26:31.215 CNTRLR   [Node 044] The node is now awake.
12:26:31.216 CNTRLR » [Node 044] Notification CC values may be stale, refreshing...
12:26:31.217 CNTRLR » [Node 044] querying notification status for Access Control...
12:26:31.391 CNTRLR   [Node 044] [Notification]
                        type:   Access Control
                        event:  Keypad lock operation
                        userId: 1
12:26:31.391 CNTRLR » [Node 044] querying notification status for Home Security...
12:26:31.818 CNTRLR » [Node 044] Sending node back to sleep...
12:26:31.835 CNTRLR   [Node 044] The node is now asleep.
```

I actually don't use my badge every day. It seems to me from this issue https://github.com/zwave-js/node-zwave-js/issues/6360, that the values are updated automatically every 6 hours. 

### Do you have a solution in mind?

Is it possible to disable forcing the CC Notification update on this module? I could also increase the wakeup by one day but we only postpone the problem which could happen at any time.

**Comment by AlCalzone** [comment]
This should be fixed since version 11.11.0. If not, please re-interview the device first. If this still doesn't fix it, please fill out this [issue template](https://github.com/zwave-js/node-zwave-js/discussions/new?category=request-support-investigate-issue).
